docs/examples/democoin/x/cool/client/cli/tx.go

```diff
@@ -46,7 +45,7 @@ func SetTrendTxCmd(cdc *codec.Codec) *cobra.Command {
 			txBldr := authtxb.NewTxBuilderFromCLI().WithCodec(cdc)
 			cliCtx := context.NewCLIContext().
 				WithCodec(cdc).
```

Comment:
Should `WithCodec` just also set the account decoder? Do we ever want a context with a new codec but the old account decoder?

Reply:
I think it should, but I'd like to open another PR with that change. Maybe just a `NewCLIContextWithCodec()` that does all this.

Reply:
ok!